static/swagger-specs/destinations.yaml

```diff
@@ -3333,6 +3344,20 @@ components:
           enum:
             - hour
             - day
+        folderNameTemplate:
+          type: string
+          title: Folder Name Template
+          description: >
+            Specify the expected folder name structure in your storage location where the exported files will be deposited. Several macros are available to use, as listed below:
+            <br><br>
+            <ul><li>`%DESTINATION%`: <b>Optional</b>. The export folder contains the destination name.</li>
+            <li>`%DATASET_NAME%`: <b>Optional</b>. The export folder contains the name of the exported segment.</li>
+            <li>`%DATASET_ID%`: <b>Optional</b>. The export folder contains the ID of the exported dataset.</li>
+            <li>`%DESTINATION_INSTANCE_ID%`: <b>Optional</b>. The export folder contains the ID of the exported dataset.</li>
+            <li>`%DESTINATION_INSTANCE_NAME%`: <b>Optional</b>. The export folder contains the ID of the exported dataset.</li>
+            <li>`%SANDBOX_NAME%`: <b>Optional</b>. The name of the sandbox environment.</li>
+            <li>`%DATETIME(YYYYMMdd_HHmmss)%` or `%EXPORT_TIME%`: <b>Optional</b>. Select one of these two options for your export folder to include the time when the datasets are exported out of Experience Platform.</li>
```

Comment:
Since these are all optional, let's put it in the introductory sentence that these are all optional, so we can save a little bit of space on each of these descriptions.